src/constants/index.js

```diff
@@ -7,6 +7,7 @@ module.exports = Object.freeze({
   LABEL_HIGH: 'High',
   LABEL_LOW: 'Low',
   LABEL_MARKET_CAP: 'Mkt Cap',
+  LABEL_PREVIOUS_CLOSE: 'Previous Close',
```

Comment:
I question the naming of this a little since it isn't really a "label".  Then again I let you make LABEL_CURRENT_PRICE a LABEL_ so I can't really complain lol.